travis/trigger-tests-in-other-repo.sh

```diff
@@ -13,13 +13,13 @@ if [ "$JOB_NUMBER" == "1" ]; then
         echo "TRAVIS_ACCESS_TOKEN_FOR_OTHER_REPO is defined. Triggering downstream repo..."
             # if this is a tagged release add that information to the dependent build request
         if [ -n "$TRAVIS_TAG" ]; then
-            ./travis/trigger-travis.sh --script "travis_wait 75 env UPSTREAM_BRANCH=$TRAVIS_BRANCH UPSTREAM_TAG=$TRAVIS_TAG ./travis/tests-unit.sh" broadinstitute viral-ngs-deploy $TRAVIS_ACCESS_TOKEN_FOR_OTHER_REPO #"UPSTREAM_BRANCH=$TRAVIS_BRANCH UPSTREAM_TAG=$TRAVIS_TAG"
+            ./travis/trigger-travis.sh --script "env UPSTREAM_BRANCH=$TRAVIS_BRANCH UPSTREAM_TAG=$TRAVIS_TAG ./travis/tests-unit.sh" broadinstitute viral-ngs-deploy $TRAVIS_ACCESS_TOKEN_FOR_OTHER_REPO #"UPSTREAM_BRANCH=$TRAVIS_BRANCH UPSTREAM_TAG=$TRAVIS_TAG"
```

Comment:
is `env UPSTREAM_BRANCH=$TRAVIS_BRANCH ./travis/tests-unit.sh` correct? Or should it be `UPSTREAM_BRANCH=$TRAVIS_BRANCH; ./travis/tests-unit.sh`?